Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -237,6 +229,17 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
         
         cell.setNeedsDisplay()
         
+        guard let url = urlString, !item.url.isAboutHomeURL else {
```

Comment:
I would break out the `let url = urlString` into it's own `guard` clause above so we don't combine the lack of urlString with showing the start page.